app/src/test/java/org/simple/clinic/selectcountry/SelectCountryUpdateTest.kt

```diff
@@ -13,23 +13,25 @@ class SelectCountryUpdateTest {
 
   private val spec = UpdateSpec(SelectCountryUpdate())
 
+  private val bangladesh = Country(
+      code = "BD",
+      endpoint = URI("https://bd.simple.org"),
+      displayName = "Bangladesh",
+      isdCode = "880"
+  )
+
+  private val countries = listOf(
+      Country(
```

Comment:
Can we extract this as a country as well just like we did with Bangaladesh?

Reply:
Done.